net-im/qutim/qutim-0.3.3_p1.ebuild

```diff
@@ -93,6 +93,7 @@ RDEPEND="${CDEPEND}
 DOCS=( AUTHORS INSTALL ChangeLog )
```

Comment:
`INSTALL` should be no longer necessary given the ebuild does all the building and install…